.github/workflows/bleeding-edge.yml

```diff
@@ -42,6 +42,4 @@ jobs:
             edm run -- python -m pip install --force-reinstall "git+http://github.com/enthought/traits.git#egg=traits"
             edm run -- python -m pip install --force-reinstall scipy
       - name: Run tests
-        uses: GabrielBB/xvfb-action@v1
-        with:
-          run: edm run -- python etstool.py test --runtime=${{ matrix.runtime }}
+        run: edm run -- python etstool.py test --runtime=${{ matrix.runtime }}
```

Comment:
You probably need to use `xvfb-run -a` here to run in a framebuffer, but I note that we're not installing either Qt or Wx so this won't actually do any GUI tests anyway.

Possibly open an issue for this?

Reply:
Right - I considered this (see the description), but we don't need a framebuffer at all for these tests, it doesn't seem worth it.

> Possibly open an issue for this?

You mean for installing Qt (or Wx) for test run purposes? Do we even _have_ any tests that exercise the GUI? Nothing appears to be being skipped in the test run.

Reply:
Hmm. I see that our PyPI-based workflow does do the Ubuntu / non-Ubuntu split, but now I'm wondering why.